guvnor-ala/guvnor-ala-spi/src/main/java/org/guvnor/ala/pipeline/execution/impl/PipelineExecutorTaskImpl.java

```diff
@@ -133,10 +142,12 @@ protected Object clone() throws CloneNotSupportedException {
                                                                       executionId);
 
         clone.setPipelineStatus(this.getPipelineStatus());
-        stageStatus.entrySet().forEach(entry -> clone.setStageStatus(entry.getKey(), entry.getValue()));
-        stageError.entrySet().forEach(entry -> clone.setStageError(entry.getKey(), entry.getValue()));
+        stageStatus.entrySet().forEach(entry -> clone.setStageStatus(entry.getKey(),
```

Comment:
yes.... and hopefully even this stageStatus.forEach(clone::setStageStatus); works...